README.md

```diff
@@ -2,102 +2,82 @@
 
 [![badge][badge]][npm-repo] [![badge-ci][badge-ci]][CircleCI]
 
-Collection of linter configurations and tools to enforce consistent code style
-and format according to Maark's recommendations.
-
-The idea is to have a structure of configurations that extend from each other
-as needed depending on the project stack, it is important to define rules to be
-able to identify anti-patterns and code-smells for specific known libraries.
-[This diagram][diagram] helps illustrate the possible configurations:
-
-```mermaid
-%%{init: {'theme':'base'}}%%
-graph TD
-    subgraph Legend
-        L1[Available]:::available --> L2[Not ready]
-    end
+[badge]: https://img.shields.io/badge/%40maarkllc%2Fcode--config-v1.1.0-blue
+[npm-repo]: https://www.npmjs.com/package/@maarkllc/code-config
 
-    A[js/base/eslint]:::available ---> B[js/react/eslint]:::available
-    A --> js/vue/eslint
-    A --> js/angular/eslint
-    B --> js/next/eslint
-    C[css/base/stylelint]:::available --> scss/base/stylelint:::available
-    A ---> D[ts/base/eslint]
-    D --> ts/angular/eslint
+[badge-ci]: https://circleci.com/gh/MAARK/code-config.svg?style=shield
+[CircleCI]: https://app.circleci.com/pipelines/github/MAARK/code-config
 
-    classDef available stroke:#0B410E,fill:#94D1BE
-```
+Collection of linter configurations and tools to enforce consistent code style
+and format according to Maark's recommendations.
 
 ## Usage
 
-### 1. Install
+Directly using `npx`
 
-```bash
-npm i --save-dev @maarkllc/code-config
+```shell
+npx @maarkllc/code-config eslint
 ```
 
-### 2. Configure [ESLint]
+Or you can install Howl-CLI globally and use `howl`
 
-Create a `.eslintrc.js` file in your project root directory and extend the
-configuration depending on the browser stack you are using.
+```shell
+npm i -g @maarkllc/code-config
 
-```javascript
-module.exports = {
-  extends: ['./node_modules/@maarkllc/code-config/js/base/eslint']
-}
+howl eslint
```

Comment:
I think we might be making the naming schema a bit complicated with this. How about we call this just `code-config` so that people who are installing this package doesn't encounter a sudden name change in the CLI.

As we discussed, this specific CLI isn't concerned with anything other than setting up this specific package.

Reply:
I will change it to be `code-config` CLI. And I will add documentation to explain more about it.